src/mlpack/core/cv/metrics/topk_accuracy_impl.hpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file topk_accuracy_impl.hpp
+ * @author Arunav Shandeelya
+ * 
+ * The implementation of class topK_accuracy_score
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_CORE_CV_METRICS_TOPK_ACCURACY_IMPL_HPP
+#define MLPACK_CORE_CV_METRICS_TOPK_ACCURACY_IMPL_HPP
+
+namespace mlpack {
+namespace cv {
+    template<typename MLAlgorithm, typename DataType, typename ResponseType, typename topK_score>
+    double topK_accuracy_score::Evaluate(MLAlgorithm& model,
+                                    const DataType& data,
+                                    const arma::Row<size_t>& labels,
+                                    const int TopK_Score& k)
+
+    {
+        if(data.n_cols! = labels.n_cols)
+        {
+            std::ostringstream fp;
+            fp << "topK_accuracy_score::Evaluate(): number of inputs ("data.n_cols")" 
```

Comment:
I think we could also print the columns for both, maybe that makes it more clear to the user. Let me know what you think.

Reply:
Ohh, I missed something here. I think you are trying to print the columns but I guess you kind of forgot `<<` operator. I think you could add them and then the build will be fixed as well.

Reply:
yeah I forgot, thanks for noticing, added.